src/view/components/Drawer/DrawerContents/index.tsx

```diff
@@ -25,10 +34,26 @@ const DrawerContents: React.StatelessComponent<OwnProps> = (props) => {
   );
 };
 
-function getDrawerItems(props: OwnProps, routes: RouteInfo[]): JSX.Element[] {
-  return routes
-    .filter(route => route.inDrawer)
-    .map(route => <DrawerItem icon={route.icon} key={route.name} name={route.name} onPress={props.onPress} />);
+function getDrawerItems(props: OwnPropsType, routesInfo: RouteInfo[]): JSX.Element[] {
```

Comment:
I thought we don't need to define return types for functions? still not sure about this too hehe

Reply:
You definitely don't have to. I just like to when I'm writing some functions because if I return the wrong type while I'm writing the function it will scream at me. Definitely not a bad practice to force a specific return type if you know exactly what you want